src/components/DiskStateProgressBar/DiskStateProgressBar.scss

```diff
@@ -23,8 +24,8 @@
     background-color: var(--progress-bar-background-color);
 
     &_compact {
+        --progress-bar-height: 12px;
```

Comment:
what the actual point? 
imho code became less readable

mb

 --progress-bar-full-height: var(--g-text-body-3-line-height);
 --progress-bar-compact-height: 12px;

and

height: var(--progress-bar-full-height);

&_compact {
    height: var(--progress-bar-compact-height);
...
}